app/src/main/java/com/myfitnesstracker/dao/IExerciseDAO.kt

```diff
@@ -1,4 +1,4 @@
-import com.myfitnesstracker.dto.ExerciseDTO
```

Comment:
Some interfaces had the interface name + DTO some didn't. I deleted DTO for more consistency

Reply:
This makes sense.  Since the class is already in a dto package, the DTO context is a given... so you don't need to add DTO to the class name as well.